include/circt/Dialect/FIRRTL/FIRRTLTypes.td

```diff
@@ -149,18 +149,18 @@ def UInt1OrUnsizedType : AnyTypeOf<[UInt1Type, UnsizedUIntType]>;
 //===----------------------------------------------------------------------===//
 
 def OneBitType : FIRRTLDialectType<
- CPred<"($_self.isa<IntType>() && $_self.cast<IntType>().getWidth() == 1) || "
-   "($_self.isa<AnalogType>() && $_self.cast<AnalogType>().getWidth() == 1)">,
+ CPred<"(firrtl::type_isa<IntType>($_self) && firrtl::type_cast<IntType>($_self).getWidth() == 1) || "
+   "(firrtl::type_isa<AnalogType>($_self) && firrtl::type_cast<AnalogType>($_self).getWidth() == 1)">,
  "UInt<1>, SInt<1>, or Analog<1>", "::circt::firrtl::FIRRTLBaseType">;
 
 
 def AnyResetType : FIRRTLDialectType<
-    CPred<"$_self.isa<FIRRTLBaseType>() && $_self.cast<FIRRTLBaseType>().isResetType()">,
+    CPred<"firrtl::type_isa<FIRRTLBaseType>($_self) && firrtl::type_cast<FIRRTLBaseType>($_self).isResetType()">,
```

Comment:
(I think this PR is inconsistent in its use of `type_isa` and `llvm::isa` for FIRRTLBaseType, since presently the two act the same for `FIRRTLBaseType` specifically (right?) and I don't think, at least not here, it's because we want to allow aliases of other kinds (that `type_isa` may support) where elsewhere we're meaning specifically to check the type hierarchy as seen by `llvm::isa`, so probably pick one and standardize on it.

Reply:
Yes, this part is inconsistent. I'll fix.